tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Logging/NLog/DirectSubmission/LogFactoryBuildLoggerConfiguration.cs

```diff
@@ -65,6 +65,32 @@ public class LogFactoryBuildLoggerConfiguration
             return CallTargetState.GetDefault();
         }
 
+        // first check to see if the configuration is null - TODO should we limit to v5.3+
```

Comment:
maybe remove TODO ?

Reply:
Thanks removed - in testing it didn't affect the earlier versions